src/components/DropZone/DropZone.tsx

```diff
@@ -171,18 +172,21 @@ export class DropZone extends React.Component<CombinedProps, State> {
     this.state = {
       type,
       id: props.id || getUniqueID(),
-      size: 'extraLarge',
+      height: Size.ExtraLarge,
       dragging: false,
       error: false,
       overlayText: translate(`Polaris.DropZone.overlayText${suffix}`),
       errorOverlayText: translate(`Polaris.DropZone.errorOverlayText${suffix}`),
       numFiles: 0,
+      width: Size.ExtraLarge,
+      loading: true,
     };
   }
 
   get getContext(): DropZoneContext {
     return {
-      size: this.state.size,
+      width: this.state.width,
```

Comment:
Since we're using state three times in this function we could deconstruct and use that sweet sweet es6 object syntax
```tsx
const {width, height, type = 'file'} = this.state;

return {width, height, type};
```